cmake/Dependencies.common.cmake

```diff
@@ -161,6 +161,10 @@ if (BUILD_FFTS)
   set(GENERATE_POSITION_INDEPENDENT_CODE ON CACHE BOOL "-fPIC")
   set(ENABLE_SHARED OFF CACHE BOOL "shared library target")
   set(ENABLE_STATIC ON CACHE BOOL "static library target")
+  # dynamic machine code generation works only for x86
+  if(ARCH)
```

Comment:
Shouldn''t it be something more specific? like `if(ARCH_ARM)` or `if(ARCH MATCHES "ARM|AARCH")'?

Reply:
Maybe use `CMAKE_SYSTEM_PROCESSOR` as in the platform/ cmake files.

Reply:
And don't bother with defining `ARCH` in the scripts.

Reply:
We may have aarch64-linux, or aarch64-qnx or anything else. I will follow @klecki suggestion.